tools/bin/release_version.sh

```diff
@@ -20,21 +22,26 @@ PART_TO_BUMP=$1
 pip install bumpversion
 bumpversion "$PART_TO_BUMP"
 
+NEW_VERSION=$(grep VERSION .env | cut -d"=" -f2)
 GIT_REVISION=$(git rev-parse HEAD)
 [[ -z "$GIT_REVISION" ]] && echo "Couldn't get the git revision..." && exit 1
 
-echo "Building and publishing version $VERSION for git revision $GIT_REVISION..."
```

Comment:
I don't know where `VERSION` is getting its value before this PR. However, it is set to the version before the bump. So this prompt is misleading.